### ISSUE TITLE
ls: write error: No space left on device error

### ISSUE THREAD

**Issue by skweebio** [ncbi/amr#45]
I faced "ls: write error: No space left on device" error when using amrfinder. 

```
Running amrfinder --plus --threads 24 -n scaffolds.fasta
AMRFinder translated nucleotide search with database /home/users/ntu/user/miniconda3/bin//data/latest
  - include -O ORGANISM, --organism ORGANISM option to add point-mutation searches
Running blastx...
ls: write error: No space left on device

*** ERROR ***
Command ls failed: status = 512

HOSTNAME: std0554
PWD: /home/users/ntu/user/scratch/work/
Progam name:  amrfinder
Command line: amrfinder --plus --threads 24 -n scaffolds.fasta
```

Currently, the input files are in the Scratch directory but the temporary directory output may be pointed to the main directory with limited storage. Will it be possible to redirect the temporary directory output to a different directory (e.g. to my scratch /tmp directory)?

**Comment by evolarjun** [comment]
Hi,

Sorry you're having this problem. I think you're correct that space on the temp device is the issue. Unfortunately right now /tmp is hard-coded in as the location for the temp files. We're working on a solution for that right now. 

Arjun

**Comment by evolarjun** [comment]
We finally got a fix (I hope) for this out. Sorry for the delay.  We just released AMRFinderPlus 3.9.8 which will respect the environment variable TMPDIR and place its temporary files there instead of /tmp if TMPDIR is set. It may take some time for the new version to percolate through bioconda if that is how you are installing AMRFinderPlus. 

Thanks, and please let us know if you continue to have this or other problems.
